src/commons/useToggleValue/index.ts

```diff
@@ -0,0 +1,23 @@
+import {useState, useRef} from 'react';
```

Comment:
There's a little mess with the "commons" path and what we export there, also after TS it got a little messier due the `commons/new` file we had to add. 
So till we sort that mess I think it will be better to move our hooks into a different hierarchy, so what about keeping it under `src/hooks` ? 

Also, I'm considering if we should export each hook as a standalone or export them under "Hooks" namespace? 
So instead of the user import like this
```
import {useToggleValue} from 'react-native-ui-lib`
```

They will 
```
import {Hooks} from 'react-native-ui-lib`
const {useToggleValue} from Hooks;
```

WDYT?

Reply:
I like it! 
On it